Dockerfile

```diff
@@ -0,0 +1,30 @@
+# vim:set ft=dockerfile:
+FROM python:3.6
+
+ENV DEBIAN_FRONTEND="noninteractive"
+
+COPY . /root/git/synapse
+RUN mkdir /syndata \
+ && apt-get update -q \
+ && apt-get install -yq --no-install-recommends \
+    build-essential \
+    cron \
+    libffi-dev \
+    libssl-dev \
+    locales \
+ && apt-get clean \
+ && apt-get purge \
+ && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
+ && locale-gen en_US.UTF-8 \
+ && dpkg-reconfigure locales \
+ && /usr/sbin/update-locale LANG=en_US.UTF-8 \
+ && cd /root/git/synapse && python setup.py install \
+ && cp synapse/docker/cortex/ram_dmon.json /syndata/dmon.json
+ENV LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8" LC_ALL="en_US.UTF-8"
```

Comment:
Do we want to plumb ``SYN_DMON_LOG_LEVEL``  to be ``WARNING`` or ``INFO`` so the base image has logging enabled by default?

Reply:
Doing this change makes it clear that the envar is available for configuring the global logging behavior.

Reply:
Ya we could do that. Should we set it to whatever the default is to show that you can change it without changing it? Should we just change the default in synapse then?

Reply:
By default the ``synapse.tools.dmon`` logging is disabled.

Reply:
I'm leaning towards keeping the image using the default setting (and also open to changing the default setting), but I would also like to set that level to `INFO` on my containers.

Reply:
My problem with the default setting is that it is a "You get nothing" setting; where we could add a default for containers to enable logging @ a ``INFO`` level.

Reply:
Added, set to WARNING